tests/rptest/tests/tls_metrics_test.py

```diff
@@ -314,39 +314,39 @@ def test_expiry_reload(self):
             f"Unexpected status after reload: {json.dumps(status_after)}"
         )
 
-    @cluster(num_nodes=3)
-    def test_crc32c(self):
-        node = self.redpanda.nodes[0]
-
-        def check_crc():
-            metrics_samples = self._get_metrics_from_node(node, ["trust_file_crc32c"])
-            assert metrics_samples is not None, "Failed to get metrics"
-            vals = self._unpack_samples(metrics_samples)["trust_file_crc32c"]
-
-            assert len(vals) > 0, "Missing crc metrics for some reason"
-
-            expected = crc32c.crc32c(
-                open(self.security.tls_provider.ca.crt, "rb").read()
-            )
-
-            for v in vals:
-                got = int(v["value"])
-                assert got == expected, f"Expected {expected}; Got {got}"
-
-            return expected
-
-        original = check_crc()
-
-        self.security.tls_provider = FaketimeTLSProvider(
-            tls=tls.TLSCertManager(self.logger, cert_expiry_days=10)
-        )
-
-        self.redpanda.set_security_settings(self.security)
-        self.redpanda.write_tls_certs()
-
-        reloaded = check_crc()
-
-        assert original != reloaded, f"Checksums unexpectedly equal"
+#    @cluster(num_nodes=3)
```

Comment:
- does @ignore not work?
- also, do we have a way to surface disabled/ignored tests? i have a scary feeling that we disable tests for various ephemeral reasons and never think about them again 😨 lol

Reply:
right. i'll use ignore instead

Reply:
I'm fine with ignoring these, but do they still work in CDT? Your changes for spot instances are only for CI right? We could keep coverage in CDT if that's not changing

Reply:
yeah. i switched to using the suite YAML file instead in the latest force-push. in that way this is executed in CDT and skipped on PRs/merges. LMK if that works